### PR TITLE
Jav 57 worker

### PR DESCRIPTION
**Pull request by seanyinx**

### CODE REVIEW

File: doorman/pom.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
   xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <parent>
     <artifactId>company</artifactId>
-    <groupId>io.servicecomb</groupId>
+    <groupId>io.servicecomb.company</groupId>
```
Comment (WillemJiang) [inline doorman/pom.xml]: We can change it to workshop as the company is not a good group name.

File: docker-run-config/pom.xml [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <groupId>io.fabric8</groupId>
+          <artifactId>docker-maven-plugin</artifactId>
+          <configuration>
+            <images>
+              <image>
+                <name>seanyinx/sc</name>
+                <alias>service-center</alias>
```
Comment (WillemJiang) [inline docker-run-config/pom.xml]: We may need to used the latest service-center for the service discovery.

**Review by WillemJiang — COMMENTED** [review]
Please add more comments on the key points those customer know about how to use ServiceComb.

File: docker-run-config/pom.xml (hunk 2 above)
Comment (seanyinx) [inline docker-run-config/pom.xml]: the one pushed by @TankTian ? it seems to have some issue.

```
[ERROR] Failed to execute goal io.fabric8:docker-maven-plugin:0.20.0:start (start) on project pojo-client: I/O Error: Unable to pull 'tank2428/servicecenter:latest': Tag latest not found in repository docker.io/tank2428/servicecenter -> [Help 1]
```

**Comment by seanyinx** [comment]
@WillemJiang java docs and comments added.

**Pull request merged**